scripts/build_fuzzers.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import subprocess
+import tarfile
+import time
+
+import toml
+
+REPO_DIR = '/home/runner/work/nearcore/nearcore/'
+G_BUCKET = 'gs://fuzzer_binaries/'
+ARCH_CONFIG_NAME = 'x86_64-unknown-linux-gnu'
+
+
+def get_archive_name(runner: str) -> str:
+    current_rev = time.strftime('%Y%m%d%H%M%S', time.gmtime())
```

Comment:
**issue:** `time.gmtime` takes the time at which it’s called. Currently, this is after `cargo fuzz build`, which can potentially take long. It could lead (especially if we build more than once a day) to archive ordering mixing up.
**todo (blocking):** move it to a `STARTUP_TIME` constant outside of the function at the top of the script, so that it more accurately reflects "commit pushed to master time"

Reply:
good idea